app/scripts/controllers/network/create-network-client.ts

```diff
@@ -20,8 +20,8 @@ import {
   SafeEventEmitterProvider,
 } from '@metamask/eth-json-rpc-provider';
 import { createInfuraMiddleware } from '@metamask/eth-json-rpc-infura';
-import type { Hex } from '@metamask/utils/dist';
-import { PollingBlockTracker } from 'eth-block-tracker/dist';
+import type { Hex } from '@metamask/utils';
```

Comment:
These changes are extra, but something I stumbled across. I think we ought to add `exports` to all of our projects to prevent this from happening. I can file an issue in the module template.

Reply:
Well, I guess we don't need these changes here though. I can extract this to a new PR.

Reply:
Removed for now: [88811d2](https://github.com/MetaMask/metamask-extension/pull/18358/commits/88811d2e189b70e0bd3b50c427486acb201e9d4f)